install/web.js

```diff
@@ -95,7 +97,7 @@ async function runWebpack() {
 
 function launchExpress(port) {
 	server = app.listen(port, () => {
-		winston.info('Web installer listening on http://%s:%s', '0.0.0.0', port);
+		winston.info('Web installer listening on http://%s:%s', ip.address(), port);
```

Comment:
This is incorrect. `express` in the wen installer is binding to `0.0.0.0`, not the external IP address.